arangod/Aql/AqlItemMatrix.cpp

```diff
@@ -296,7 +295,7 @@ auto AqlItemMatrix::RowIterator::hasMore() const noexcept -> bool {
   TRI_ASSERT((_matrix != nullptr && _blockIndex < _matrix->numberOfBlocks()) ||
              _rowIndex == 0);
   // If _blockIndex is valid, _rowIndex must be, too.
-  return ADB_LIKELY(_matrix != nullptr) &&
+  return ADB_LIKELY_DEPRECATED(_matrix != nullptr) &&
```

Comment:
Instead of using the deprecated macro, can we remove the annotation here completely? Sounds better than supporting the deprecated macro.